.github/workflows/pull_requests.yaml

```diff
@@ -70,6 +72,8 @@ jobs:
         with:
           go-version: '1.20'
           cache: true
+      - name: Integration test merged buckets
+        run: ENABLE_MERGED_PROPERTY_BUCKETS=true ./test/run.sh --integration-only
```

Comment:
I don't like the idea of passing special environment variables to our test scripts. Now if I want to run integration tests I have to remember to run tests without and with special environment variable `ENABLE_MERGED_PROPERTY_BUCKETS`. We didn't have to modify our test scripts when we were switching to roaring bitmaps (and this was the same type of change) and I'm wondering if we now have to do it like that?
Can we create separate unit / integration / e2e tests that would test both scenarios? or can we have a dedicated chaos pipeline test that would do this?